tests/test_queries.py

```diff
@@ -51,6 +52,32 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="module")
+def la_expected_electricity_hour_16():
```

Comment:
I thought you could use `tmp_path` to create temp dir in pytest?

Reply:
`tmp_path` has function level scope and couldn't be used here. I was lazy. Apparently, there is `tmp_path_factory` which can be used for session and module level scope. I'll fix it.